test/integration/reports_show_test.rb

```diff
@@ -1,16 +1,16 @@
-require 'test_helper'
+require "test_helper"
 
-class ReportShowTest < ActionDispatch::IntegrationTest
-  def test_slug
-    data = <<-DATA
-[{
-  "name": "test",
-  "ips": 10.1,
-  "stddev": 0.3,
-  "microseconds": 3322,
-  "iterations": 221,
-  "cycles": 16
-}]
+class RoutingTest < ActionDispatch::IntegrationTest
+  def test_short_id
+    data = <<~DATA
+      [{
+        "name": "test",
+        "ips": 10.1,
+        "stddev": 0.3,
+        "microseconds": 3322,
+        "iterations": 221,
+        "cycles": 16
+      }]
```

Comment:
@bronzdoc Please make sure not to roll back my changes. We are now using `slug` instead of `short_id`. Also, this is a `ReportShowTest` not a `RoutingTest`.